src/styles/blocks/contact-us.scss

```diff
@@ -0,0 +1,36 @@
+.contact-us {
+  box-sizing: border-box;
+
+  &__form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__input {
+    font-family: "Manrope", sans-serif;
+    box-sizing: border-box;
+    width: 100%;
+    height: 48px;
+    padding: 14px 16px;
+    border-radius: 8px;
+    border: 1px solid transparent;
+    margin-top: 16px;
+
+    background-color: #f9f9f9;
+
+    &--message {
+      height: 120px;
+      resize: none;
+    }
+    @include hover (border, 1px solid #DBDBDB);
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new `border` to the element on hover. Add default transparent border of the same width, and change its color on `:hover`

Reply:
Please fix this comment